packages/nimbus/src/components/data-table/components/data-table.column.tsx

```diff
@@ -0,0 +1,72 @@
+import { ColumnResizer, Column as RaColumn } from "react-aria-components";
+import { ArrowDownward } from "@commercetools/nimbus-icons";
+import { Divider, Flex } from "@/components";
+import { useDataTableContext } from "./data-table.context";
+import {
+  DataTableColumnSlot,
+  DataTableHeaderSortIcon,
+  DataTableColumnResizer,
+} from "../data-table.slots";
+import type { DataTableColumnComponent } from "../data-table.types";
+
+export const DataTableColumn: DataTableColumnComponent = ({
+  children,
+  ref,
+  column,
+  unstyled,
+  isInternalColumn,
+  tabIndex,
+  ...otherProps
+}) => {
+  const { sortDescriptor } = useDataTableContext();
+  const isActive = sortDescriptor?.column === column?.id;
+  /**
+   * TODO: Pass styles to DataTableColumnSlot from props,
+   * except for height/minHeight/maxHeight which enable react-aria's
+   * column resizing
+   */
+  return (
+    <DataTableColumnSlot unstyled={unstyled} tabIndex={tabIndex} asChild>
```

Comment:
Probably a dumb question, but why do we allow for `unstyled` to be supported explicitly?